src/scripts/InstanceScripts/Wotlk/Raid_IceCrownCitadel.cpp

```diff
@@ -442,8 +415,29 @@ class IceCrownCitadelScript : public InstanceScript
         case ACTION_BATTLE_EVENT:
             scriptEvents.addEvent(EVENT_WIPE_CHECK, 5000);
             setData(DATA_GUNSHIP_EVENT, InProgress);
-            SendMusicToPlayers(17289);
-            sEventMgr.AddEvent(static_cast<Object*>(skybreaker), &Object::PlaySoundToSet, (uint32_t)17289, EVENT_UNK, 41000, 1, EVENT_FLAG_DO_NOT_EXECUTE_IN_WORLD_CONTEXT);
+            break;
+        case ACTION_BATTLE_DONE:
+            if (getData(DATA_GUNSHIP_EVENT) == Finished)
+            {
+                if (TeamInInstance == TEAM_ALLIANCE && skybreaker)
+                {
+                    skybreaker->EnableMovement(true, mInstance);
```

Comment:
technicaly you cannot win without an own transport thats why i dont checked it there ^^ but i wil add it

Reply:
that may be true. All I see is a function where you sometimes check if the pointer is nullptr and sometimes not. Every static analysy tool will tell you, that it could be nullptr. We should, especially in the case of transports, check if the pointer is still valid

Reply:
nevermind, you already did that in line 422 ;)